src/Register.js

```diff
@@ -1,16 +1,25 @@
 import React, { useState } from 'react'
 import './register.css'
 
-const Register = ({handleRegister}) => {
+const Register = ({handleRegister, registerStatus}) => {
     const [username, setUsername] = useState("")
     const [email, setEmail] = useState("")
     const [password, setPassword] = useState("")
-    const [registerStatus, setRegisterStatus] = useState("")
+    const [passwordStatus, setPasswordStatus] = useState("")
     const isValidPassword = (e) => {
         var pw = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,25}$/.test(e.target.value)
-        pw ? setRegisterStatus("") && setPassword(pw) : setRegisterStatus("Password needs to be 8-25 length, at least one uppercase, one lowercase, one number");
-        console.log(pw)
+        if(pw || e.target.value == ""){
```

Comment:
It's probably a typo, but we should use "===" instead of "==" for strict comparison. The difference is "==" compares by converting both values to a common type and compare which is error-prone, while "===" takes their types into consideration.

Reply:
noted